docs/user/index.md

```diff
@@ -66,6 +66,25 @@ You can do this by running the following command:
 conda auth logout <channel_name>
 ```
 
+### Storage backend unavailable?
+
+Conda auth relies on the [keyring](https://github.com/jaraco/keyring) package to store its passwords and secrets.
+Because of this, it only supports a limited number of operating systems, mostly desktop operating systems like
+Windows, OSX and several Linux variants.
+
+If you want to use conda-auth, but are not using a supported operating system, you can install the
+[keyring-alt](https://github.com/jaraco/keyrings.alt) package:
+
+```
+conda install -c conda-forge keyrings.alt
+```
```

Comment:
Does this suffice to enable the use yet?

Reply:
Yes, when this package is installed, the keyring package detects that and then has a fallback to use for all platforms.